plugin/cache/README.md

```diff
@@ -50,6 +51,11 @@ cache [TTL] [ZONES...] {
   **DURATION** defaults to 1m. Prefetching will happen when the TTL drops below **PERCENTAGE**,
   which defaults to `10%`, or latest 1 second before TTL expiration. Values should be in the range `[10%, 90%]`.
   Note the percent sign is mandatory. **PERCENTAGE** is treated as an `int`.
+* `serve_expired`, when serve\_expired is set, CoreDNS always will serve an expired cache to a client if an
+  expired cache entry exists. When this happens, CoreDNS will attempt to refresh the cache entry after sending
+  the expired cache entry to the client. The responses have a TTL of 0. **DURATION** is how far back to consider
+  stale responses as fresh. The default is: 'serve\_expired no 1h'. Hint: the bigger the **CAPACITY** the more
```

Comment:
I would remove the hint, sounds a bit pedantic

Reply:
This is something I would have never thought of without external feedback, but you are right. Removed. Thanks.